api-reference/v1.0/api/user_list.md

```diff
@@ -86,6 +86,42 @@ Content-length: 608
 }
 ```
 
+If you need a different property set, you can use the ODATA `$select` query parameter. For example, to return _displayName_, _givenName_, _id_ and _postalCode_, you would use the add the following to your query `$select=displayName,givenName,postalCode`
```

Comment:
First comment "OData" not "ODATA".

Second thing is that for list_users there are 2 reasons that you might use $select:

1. To get properties that are not in the default set.
2. To limit the properties in the response to the set needed by your app in order to reduce network load.

I like that the first point is covered in the Query parameters section, I wonder if a short discussion of the second belongs there as well. We are going to put a generic version of this guidance in a best practices topic -- but I think the most pertinent resources are probably users, groups, messages, events, and maybe files and I don't think it would hurt to explicitly call it out in the "list" topics for them.
NOTE: This guidance about reducing network traffic should probably also be added to the v1.0 topic. Also -- I saw other instances of "ODATA" this PR.